packages/scripts/build-package.ts

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env -S node --experimental-specifier-resolution=node --loader=tsx
```

Comment:
we have been running bin scripts always with tsx so far using tsx from scripts, I would love a consistency